Library/OcBootManagementLib/ImageLoader.c

```diff
@@ -969,7 +969,7 @@ InternalEfiLoadImage (
   // Load the image ourselves in secure boot mode.
   //
   if (SecureBootStatus == EFI_SUCCESS) {
-    if ((SourceBuffer != NULL) && (OcImageLoaderCaps == NULL)) {
+    if (SourceBuffer != NULL) {
```

Comment:
Do you remember what was the purpose for `OcImageLoaderCaps == NULL` check?

Reply:
Yes. Go to the commit I'm (proposing) reverting a small part of: https://github.com/acidanthera/OpenCorePkg/commit/125d8e8df36bd7ca12085d9670dca9060cf49c76

And forward search in the page for 'fat slice'. This leads to changes I made (see lines 944 and 958 of the revised ImageLoader.c file in the result of that diff). I am reverting to your original code.

I believe I added the fat slice check because it seemed to me (wrongly, it now turns out) that there was no way that a fat slice could sanely have success in secure boot status (since they date from before signing).

Reply:
Got it, ok.